docs/apis/streaming/connectors/kinesis.md

```diff
@@ -26,7 +26,7 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-The Kinesis connector provides access to [Amazon AWS Kinesis Streams](http://aws.amazon.com/kinesis/streams/). 
```

Comment:
Just a small nit note: we should try to keep comment / doc reformatting to a minimal unless necessary, since it'll make the diff quite unreadable. You can also take a look at the code contribution / style guidelines here: https://flink.apache.org/contribute-code.html :)